package.json

```diff
@@ -4,43 +4,125 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "build": "wp-scripts build && npm run theme",
-    "format:js": "wp-scripts format-js",
-    "lint:js": "wp-scripts lint-js",
-    "packages-update": "wp-scripts packages-update",
-    "start": "wp-scripts start",
+    "build": "wds-build && npm run theme",
+    "format": "wds-format-js && wds-format-styles && npm run format:prettier",
+    "format:prettier": "[[ -f .prettierignore ]] && ./node_modules/.bin/prettier ./ --write",
```

Comment:
Can I get an Amen!? We do not use Prettier for JS or Styles, but everything else...go forth!